Subject: Correct the KVI_SOURCES_DATE{,_NUMERIC}.
Origin: upstream, https://svn.kvirc.de/kvirc/changeset/4552
Last-Update: 2010-06-28
---
 src/kvilib/config/kvi_sourcesdate.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/src/kvilib/config/kvi_sourcesdate.h
+++ b/src/kvilib/config/kvi_sourcesdate.h
@@ -36,8 +36,8 @@
 * \def KVI_SOURCES_DATE Defines the sources date
 * \def KVI_SOURCES_DATE_NUMERIC Defines the sources date
 */
-#define KVI_SOURCES_DATE "20100519"
-#define KVI_SOURCES_DATE_NUMERIC 0x20100519
+#define KVI_SOURCES_DATE "20100627"
+#define KVI_SOURCES_DATE_NUMERIC 0x20100627
 
 /**
 * \brief This is the date of the sources before that we should force a setup
