diff -u admin/acinclude.m4.in.2 admin/acinclude.m4.in
--- admin/acinclude.m4.in.2	2008-03-30 18:39:06.000000000 +0200
+++ admin/acinclude.m4.in	2008-04-15 09:48:33.000000000 +0200
@@ -1105,7 +1105,7 @@
 			ss_save_LIBS="$LIBS"
 			CPPFLAGS="-I$SS_X_INCDIR -I$SS_QT3_INCDIR -I$SS_KDE_INCDIR $CPPFLAGS $X_CFLAGS"
 			CXXFLAGS="-O2 -Wall $CXXFLAGS"
-			LIBS="-L$SS_KDE_LIBDIR -L$SS_X_LIBDIR $SS_X_LDFLAGS $SS_X_LIBLINK -L$SS_QT3_LIBDIR $LIBS -lsoundserver_idl -lkdecore -l$SS_QTLIB_NAME -Wl,-rpath $SS_KDE_LIBDIR"
+			LIBS="-L$SS_KDE_LIBDIR -L$SS_X_LIBDIR $SS_X_LDFLAGS $SS_X_LIBLINK -L$SS_QT3_LIBDIR $LIBS -lsoundserver_idl -lkdecore -l$SS_QTLIB_NAME"
 			if test "$SS_LINK_TO_LIBDL" = "yes"; then
 				LIBS="$LIBS -ldl";
 			fi
@@ -2252,7 +2252,7 @@
 		if test -n "$SS_X_LIBLINK"; then
 			LIBS="$LIBS $SS_X_LIBLINK"
 		fi
-		LIBS="-L$SS_KDE_LIBDIR -L$SS_QT3_LIBDIR -lkdecore -l$SS_QTLIB_NAME $LIBS -Wl,-rpath $SS_KDE_LIBDIR"
+		LIBS="-L$SS_KDE_LIBDIR -L$SS_QT3_LIBDIR -lkdecore -l$SS_QTLIB_NAME $LIBS"
 		if test "$SS_LINK_TO_LIBDL" = "yes"; then
 			LIBS="$LIBS -ldl";
 		fi
@@ -2505,6 +2505,7 @@
 	AC_SUBST(SS_CPPFLAGS)
 	AC_SUBST(SS_INCDIRS)
 	AC_SUBST(SS_LDFLAGS)
+	SS_RPATH=""
 	AC_SUBST(SS_RPATH)
 	AC_SUBST(SS_LIBDIRS)
 	AC_SUBST(SS_LIBLINK)
