Subject: Fix upstream bug #817.
Origin: upstream, https://svn.kvirc.de/kvirc/changeset/4569
Last-Update: 2010-07-01
---
 src/modules/options/dialog.cpp |    1 +
 1 file changed, 1 insertion(+)

Index: b/src/modules/options/dialog.cpp
===================================================================
--- a/src/modules/options/dialog.cpp
+++ b/src/modules/options/dialog.cpp
@@ -178,6 +178,7 @@ KviOptionsDialog::KviOptionsDialog(QWidg
 	m_pTreeWidget->header()->hide();
 	m_pTreeWidget->setRootIsDecorated(true);
 	m_pTreeWidget->setSortingEnabled(false);
+	m_pTreeWidget->setSizePolicy(QSizePolicy::MinimumExpanding,QSizePolicy::MinimumExpanding);
 
 	connect(m_pTreeWidget,SIGNAL(currentItemChanged(QTreeWidgetItem*, QTreeWidgetItem *)),this,SLOT(treeWidgetItemSelectionChanged(QTreeWidgetItem *, QTreeWidgetItem *)));
 
