diff -Nru src/kvirc/kvs/kvi_kvs_coresimplecommands_sz.cpp src/kvirc/kvs/kvi_kvs_coresimplecommands_sz.cpp
--- src/kvirc/kvs/kvi_kvs_coresimplecommands_sz.cpp	2006-07-28 14:51:38.000000000 +0200
+++ src/kvirc/kvs/kvi_kvs_coresimplecommands_sz.cpp	2008-01-28 09:43:06.293187308 +0100
@@ -38,6 +38,7 @@
 #include "kvi_netutils.h"
 #include "kvi_menubar.h"
 #include "kvi_userinput.h"
+#include <cstdlib>
 
 namespace KviKvsCoreSimpleCommands
 {
