Subject: Update the MOTD. (Upstream r3902)
Author: Elvio Basello <hell@hellvis69.netsons.org>
Origin: upstream, https://svn.kvirc.de/kvirc/changeset/3902?format=diff&new=3902
Last-Update: 2010-01-27
---
 src/kvilib/config/kvi_defaults.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: b/src/kvilib/config/kvi_defaults.h
===================================================================
--- a/src/kvilib/config/kvi_defaults.h
+++ b/src/kvilib/config/kvi_defaults.h
@@ -70,7 +70,7 @@
 #define KVI_DEFAULT_USERNAME "kvirc"
 #define KVI_DEFAULT_REALNAME "KVIrc " KVI_VERSION " '" KVI_RELEASE_NAME "' http://kvirc.net/"
 
-#define KVI_DEFAULT_PART_MESSAGE "No matter how dark the night, somehow the Sun rises once again"
+#define KVI_DEFAULT_PART_MESSAGE "Once you know what it is you want to be true, instinct is a very useful device for enabling you to know that it is"
 #define KVI_DEFAULT_QUIT_MESSAGE "KVIrc $version http://www.kvirc.net/"
 #define KVI_DEFAULT_CTCP_USERINFO_REPLY "I'm too lazy to edit this field."
 #define KVI_DEFAULT_CTCP_PAGE_REPLY "Your message has been received and logged"
