Index: kvirc-4.0.0~svn3039/data/icons/128x128/CMakeLists.txt
===================================================================
--- kvirc-4.0.0~svn3039.orig/data/icons/128x128/CMakeLists.txt	2009-02-25 05:52:14.264060119 +0100
+++ kvirc-4.0.0~svn3039/data/icons/128x128/CMakeLists.txt	2009-02-25 05:52:25.833220274 +0100
@@ -7,7 +7,7 @@
 		INSTALL(FILES ${files} DESTINATION ${CMAKE_INSTALL_PREFIX}/Contents/Resources/icons/128x128/)
 	ELSE()
 		# Assume linux
-		INSTALL(FILES ${files} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/128x128/)
+		INSTALL(FILES ${files} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/128x128/apps/)
 	ENDIF()
 ELSEIF(WIN32)
 	INSTALL(FILES ${files} DESTINATION ${CMAKE_INSTALL_PREFIX}/icons/128x128/)
Index: kvirc-4.0.0~svn3039/data/icons/16x16/CMakeLists.txt
===================================================================
--- kvirc-4.0.0~svn3039.orig/data/icons/16x16/CMakeLists.txt	2009-02-25 05:52:42.420060756 +0100
+++ kvirc-4.0.0~svn3039/data/icons/16x16/CMakeLists.txt	2009-02-25 05:52:52.369190670 +0100
@@ -7,7 +7,7 @@
 		INSTALL(FILES ${files} DESTINATION ${CMAKE_INSTALL_PREFIX}/Contents/Resources/icons/16x16/)
 	ELSE()
 		# Assume linux
-		INSTALL(FILES ${files} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/16x16/)
+		INSTALL(FILES ${files} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/16x16/apps/)
 	ENDIF()
 ELSEIF(WIN32)
 	INSTALL(FILES ${files} DESTINATION ${CMAKE_INSTALL_PREFIX}/icons/16x16/)
Index: kvirc-4.0.0~svn3039/data/icons/64x64/CMakeLists.txt
===================================================================
--- kvirc-4.0.0~svn3039.orig/data/icons/64x64/CMakeLists.txt	2009-02-25 05:53:04.516061215 +0100
+++ kvirc-4.0.0~svn3039/data/icons/64x64/CMakeLists.txt	2009-02-25 05:53:16.377163736 +0100
@@ -7,7 +7,7 @@
 		INSTALL(FILES ${files} DESTINATION ${CMAKE_INSTALL_PREFIX}/Contents/Resources/icons/64x64/)
 	ELSE()
 		# Assume linux
-		INSTALL(FILES ${files} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/64x64/)
+		INSTALL(FILES ${files} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/64x64/apps/)
 	ENDIF()
 ELSEIF(WIN32)
 	INSTALL(FILES ${files} DESTINATION ${CMAKE_INSTALL_PREFIX}/icons/64x64/)
Index: kvirc-4.0.0~svn3039/data/icons/scalable/CMakeLists.txt
===================================================================
--- kvirc-4.0.0~svn3039.orig/data/icons/scalable/CMakeLists.txt	2009-02-25 05:53:26.968060705 +0100
+++ kvirc-4.0.0~svn3039/data/icons/scalable/CMakeLists.txt	2009-02-25 05:53:38.248719305 +0100
@@ -7,7 +7,7 @@
 		INSTALL(FILES ${files} DESTINATION ${CMAKE_INSTALL_PREFIX}/Contents/Resources/icons/scalable/)
 	ELSE()
 		# Assume linux
-		INSTALL(FILES ${files} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/scalable/)
+		INSTALL(FILES ${files} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/scalable/apps/)
 	ENDIF()
 ELSEIF(WIN32)
 	INSTALL(FILES ${files} DESTINATION ${CMAKE_INSTALL_PREFIX}/icons/scalable/)
